content/engine/install/_index.md

```diff
@@ -41,16 +41,18 @@ through Docker Desktop. For instructions on how to install Docker Desktop, see:
 
```

Comment:
We should add the note here Vanessa mentioned.  Something like "For Docker Desktop on RHEL please join our [Early Access Program](https://www.docker.com/docker-desktop-preview-program/)."

Reply:
I put it on the rhel installation page (here seems like the wrong place, we don't call out specific distro versions, only OS versions)